src/utils/assets.js

```diff
@@ -151,11 +152,34 @@ export const addressesEqual = (address1: ?string, address2: ?string): boolean =>
 };
 
 export const getAssetData = (
-  assetsData: Asset[],
+  userAssets: Asset[],
   supportedAssetsData: Asset[],
   assetSymbol: string,
 ): Asset | Object => {
-  return assetsData.find(({ symbol }: Asset) => symbol === assetSymbol)
+  return userAssets.find(({ symbol }: Asset) => symbol === assetSymbol)
   || supportedAssetsData.find(({ symbol }: Asset) => symbol === assetSymbol)
   || {};
 };
+
+export const getAssetDataByAddress = (
```

Comment:
I'd avoid exporting symbols that are only used on the same file.

Reply:
What do you mean? It's for checking against method argument.

Reply:
He doesn't like the `export` part

Reply:
Sorry, I didn't explain this because I mentioned it on another comment.

When linters search for unused symbols, they don't go through all of the codebase, so they don't check if exported symbols are being used somewhere else.

This means that if in the future, we remove the code that uses this function, we'd have to manually search for it on the rest for the project, or just leave it there unused. Unused code will have to be maintained just because it's already there, or since it's not being used we won't know if it's broken.

I think we should help linters catch issues like this and others, before they get into the upstream.